servers/visual/visual_server_scene.cpp

```diff
@@ -2732,7 +2781,14 @@ bool VisualServerScene::_light_instance_update_shadow(Instance *p_instance, cons
 					ortho_transform.basis = transform.basis;
 					ortho_transform.origin = x_vec * (x_min_cam + half_x) + y_vec * (y_min_cam + half_y) + z_vec * z_max;
 
-					VSG::scene_render->light_instance_set_shadow_transform(light->instance, ortho_camera, ortho_transform, 0, distances[i + 1], i, bias_scale);
+					VSG::scene_render->light_instance_set_shadow_transform(
+							light->instance,
+							ortho_camera,
+							ortho_transform,
+							z_max - z_min_cam,
+							distances[i + 1],
+							i,
+							radius * 2.0 / texture_size);
```

Comment:
This doesn't seem anything like the bias scale from before (maybe intentional, but wanted to check this was correct).

In master this value is `shadow_texel_size`, and bias_scale is `z_max - z_min_cam`.